config/development.json

```diff
@@ -66,6 +66,7 @@
 		"help": true,
 		"help/courses": true,
 		"hosting": true,
+		"hosting/all-sites": true,
```

Comment:
Maybe keep this off, I think we confused other devs last time. we can enable with ?flags=hosting/all-sites or starting with `ENABLE_FEATURES=hosting/all-sites npm start`

Reply:
ce07082f575cac16bd6ba4cc93069cf2db53c7f6